main.go

```diff
@@ -49,6 +52,20 @@ func main() {
 
 	ctx, cancel := context.WithCancel(context.Background())
 
+	// Minimum severity of the messages that the logger will log:
+	// info, warning, error and fatal
+	logCfg := logger.Config{
+		Level:     "info",
+		Component: os.Getenv("FLY_APP_NAME"),
```

Comment:
if this needs to be set, consider using `panic("FLY_APP_NAME env var must be set")` if the value is `""`